client/src/app/components/Projects/Filters/Dropdown/Dropdown.js

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from "react";
+import Button from "../Button/Button";
+import classNames from "classnames";
+import styles from "./Dropdown.module.css";
+
+class Dropdown extends Component {
+  state = {
+    areOptionsShown: false,
+    selectedOptions: []
+  };
+
+  containerRef = React.createRef();
```

Comment:
Just a typo - container?

Reply:
What do you mean ? For the `containerRef` ? It looks fine to me. This is being used for getting a reference to the DOM element